packages/jsii-reflect/lib/util.ts

```diff
@@ -1,7 +1,79 @@
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
 export function indexBy<T>(xs: T[], f: (x: T) => string): { [key: string]: T } {
   const ret: { [key: string]: T } = {};
   for (const x of xs) {
     ret[f(x)] = x;
   }
   return ret;
 }
+
+/**
+ * Find the directory that contains a given dependency, identified by its 'package.json', from a starting search directory
+ */
+export async function findDependencyDirectory(
```

Comment:
Is this implementation a copy of the one in `jsii-pacmak`? I wouldn't mind a `@jsii/utils` package...

Reply:
Yeeeeaaa -- hhh... I don't disagree, but I didn't want to make too many changes at once